docs/pages/experiments/docs/demos.md

```diff
@@ -1,19 +1,27 @@
 # Demos
 
-<p class="description">Demos</p>
+<p class="description">The different variants of demo containers we have in the docs.</p>
 
 ## Standard demo
 
 {{"demo": "DemoInDocs.js"}}
 
-## "bg": "inline" demo
+## "bg": "inline"
 
 {{"demo": "DemoInDocs.js", "bg": "inline"}}
 
+## "bg": "outlined"
```

Comment:
I think it's all good because "standard" is actually just the state when no bg is explicitly defined, which then renders the `outlined` variant. You can't do `"bg": "standard"`. We're left with a similar structure to what you propose: `outlined`, `"bg": true`, and `gradient`.

I think that for most cases, `outlined` fits better as the default, though. Wouldn't change it right now.